### PR TITLE
[Windows] Add MSVC++ 14.32

### PR DESCRIPTION
**Pull request by al-cheb**
# Description
- Add MSVC++ 14.32

#### Related issue:
https://github.com/actions/virtual-environments/issues/5910

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by mikhailkoliada — DISMISSED** [review]

**Review by miketimofeev — APPROVED** [review]

File: images/win/toolsets/toolset-2022.json [hunk 1]
```diff
@@ -242,6 +242,14 @@
...
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.MFC",
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.MFC.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64.Spectre",
```
[the remark this answers is not in the document]
Comment (al-cheb) [reply images/win/toolsets/toolset-2022.json]: Fixed. Thank you.

File: images/win/toolsets/toolset-2022.json [hunk 2]
```diff
@@ -242,6 +242,14 @@
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.MFC",
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.MFC.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64",
```
[the remark this answers is not in the document]
Comment (al-cheb) [reply images/win/toolsets/toolset-2022.json:249]: Fixed.

**Review by miketimofeev — DISMISSED** [review]

File: images/win/toolsets/toolset-2022.json [hunk 3]
```diff
@@ -242,14 +242,16 @@
...
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ARM.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ARM64",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ARM64.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.Spectre",
-            "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64",
```
Comment (dangrif) [inline images/win/toolsets/toolset-2022.json]: For ARM/ARM64 targeting, you'll probably also want to add:

            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM.Spectre", 
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64.Spectre",               
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64EC",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64EC.Spectre",               
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM.Spectre",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64.Spectre",               
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64EC",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64EC.Spectre",

Comment (al-cheb) [reply images/win/toolsets/toolset-2022.json]: Updated. Thank you.

**Review by miketimofeev — APPROVED** [review]

File: images/win/toolsets/toolset-2022.json (hunk 3 above)
Comment (al-cheb) [reply images/win/toolsets/toolset-2022.json]: There are no Microsoft.VisualStudio.Component.VC.14.32.17.2.X.ARM64EC components for VS 2022 - https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise?view=vs-2022

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**